### PR TITLE
Dịch introduction/index_vn.md phần 7

### LINKED ISSUE #105: Dịch introduction/index_vn.md phần 7
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_introduction/index_vn.md#L452-L525

### PR DESCRIPTION
**Pull request by lkhphuc**
## Pull Request checklist

**Đánh dấu tick (x, không có dấu cách xung quanh) vào các mục dưới đây**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #105 

* [x] Bản dịch này có bám sát [Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md) không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm
    * Tiêu đề (số lượng dấu `#` đầu dòng)
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`)
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt)
    * Không dịch các phần code (nằm giữa hai dấu `````)
    * Copy các công thức toán từ bản gốc (các đoạn có `$`)
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`)
    * Không tự thêm bớt các dòng trắng
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm [hướng dẫn hoặc yêu cầu dịch hình](img/README.md).

* [x] Điền tên của bạn và vào mục "Những người thực hiện" ở cuối file.

* [x] Pull Request này đã sẵn sàng cho việc Review chưa?

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này.

## Với reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_introduction/index_vn.md [hunk 1]
```diff
@@ -594,14 +594,18 @@ On our way to discussing deep neural networks,
 we will discuss some more traditional methods.
 -->
 
-*dịch đoạn phía trên*
-
+Phần lớn học máy đều liên quan đến *biến đổi* dữ liệu theo một cách đó.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Phần lớn học máy đều liên quan đến *biến đổi* dữ liệu theo một cách nào đó.
```

File: chapter_introduction/index_vn.md [hunk 2]
```diff
@@ -594,14 +594,18 @@ On our way to discussing deep neural networks,
...
+Đó có thể là một hệ thống nhận ảnh đầu vào và dự đoán bức ảnh có chứa khuôn mặt cười hay không.
+Hoặc đó cũng có thể là một hệ thống nhận vào những dữ liệu đo đạc từ cảm biến và
+dự đoán xem những số liệu đó là *bình thường* hay *bất thường*.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]: Mỗi câu trong một dòng cho dễ review em ơi. Nếu để cách dòng thì qua vài cái comment là phải kéo tít mù mới thấy phần trước hoặc sau của câu.

File: chapter_introduction/index_vn.md [hunk 3]
```diff
@@ -594,14 +594,18 @@ On our way to discussing deep neural networks,
 we will discuss some more traditional methods.
 -->
 
-*dịch đoạn phía trên*
-
+Phần lớn học máy đều liên quan đến *biến đổi* dữ liệu theo một cách đó.
+Đó có thể là một hệ thống nhận ảnh đầu vào và dự đoán bức ảnh có chứa khuôn mặt cười hay không.
+Hoặc đó cũng có thể là một hệ thống nhận vào những dữ liệu đo đạc từ cảm biến và
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Hoặc đó cũng có thể là một hệ thống nhận vào dữ liệu đo đạc từ cảm biến và
```

File: chapter_introduction/index_vn.md [hunk 4]
```diff
@@ -594,14 +594,18 @@ On our way to discussing deep neural networks,
...
+dự đoán xem những số liệu đó là *bình thường* hay *bất thường*.
+Ở đây chúng ta gọi *mô hình* là một hệ thống tính toán một dạng dữ liệu đầu vào
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Ở đây chúng ta gọi *mô hình* là một hệ thống tính toán lấy đầu vào là một dạng dữ liệu và trả về kết quả dự đoán có thể ở một dạng dữ liệu khác.
```

File: chapter_introduction/index_vn.md [hunk 5]
```diff
@@ -594,14 +594,18 @@ On our way to discussing deep neural networks,
...
 -->
 
-*dịch đoạn phía trên*
-
+Phần lớn học máy đều liên quan đến *biến đổi* dữ liệu theo một cách đó.
+Đó có thể là một hệ thống nhận ảnh đầu vào và dự đoán bức ảnh có chứa khuôn mặt cười hay không.
+Hoặc đó cũng có thể là một hệ thống nhận vào những dữ liệu đo đạc từ cảm biến và
+dự đoán xem những số liệu đó là *bình thường* hay *bất thường*.
+Ở đây chúng ta gọi *mô hình* là một hệ thống tính toán một dạng dữ liệu đầu vào 
+và trả ra những kết quả dự đoán mà có thể ở một dạng khác.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
```

File: chapter_introduction/index_vn.md [hunk 6]
```diff
@@ -612,7 +616,11 @@ and some people might disagree on whether the proposed update
 constituted an improvement or a decline.
 -->
 
-*dịch đoạn phía trên*
+Trước đây, chúng tôi giới thiệu học máy theo kiểu "học từ kinh nghiệm".
+*Học* ở đây tức là có *tiến bộ* qua thời gian với một bài toán nào đó.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
*Học* ở đây tức là có *tiến bộ* ở một tác vụ nào đó theo thời gian.
```

File: chapter_introduction/index_vn.md [hunk 7]
```diff
@@ -612,7 +616,11 @@ and some people might disagree on whether the proposed update
...
+Nhưng ai sẽ có nhiệm vụ chỉ ra như thế nào là tiến bộ?
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Nhưng ai sẽ chỉ ra như thế nào là tiến bộ?
```

File: chapter_introduction/index_vn.md [hunk 8]
```diff
@@ -612,7 +616,11 @@ and some people might disagree on whether the proposed update
...
+Giả sử chúng ta đề xuất một bản cập nhật cho mô hình nhưng sẽ có người không đồng ý rằng
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Thử tưởng tượng bạn đang đề xuất cập nhật mô hình nhưng một số người có thể không đồng ý liệu việc đó giúp cải thiện mô hình hay không.
```

File: chapter_introduction/index_vn.md [hunk 9]
```diff
@@ -612,7 +616,11 @@ and some people might disagree on whether the proposed update
 constituted an improvement or a decline.
 -->
 
-*dịch đoạn phía trên*
+Trước đây, chúng tôi giới thiệu học máy theo kiểu "học từ kinh nghiệm".
+*Học* ở đây tức là có *tiến bộ* qua thời gian với một bài toán nào đó.
+Nhưng ai sẽ có nhiệm vụ chỉ ra như thế nào là tiến bộ?
+Giả sử chúng ta đề xuất một bản cập nhật cho mô hình nhưng sẽ có người không đồng ý rằng 
+bản cập nhật của chúng ta là cải tiến hay lại đang cải lùi.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
```

File: chapter_introduction/index_vn.md [hunk 10]
```diff
@@ -629,7 +637,14 @@ Because lower is better, these functions are sometimes called
 *loss functions* or *cost functions*.
 -->
 
-*dịch đoạn phía trên*
+Để có thể phát triển một mô hình toán học chính quy cho học máy,
+chúng ta cần những phương pháp đo đạc chính quy xem mô hình của mình đang tốt (hoặc tệ) như thế nào.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
chúng ta cần những phép đo chính quy xem mô hình đang tốt (hoặc tệ) như thế nào.
```

File: chapter_introduction/index_vn.md [hunk 11]
```diff
@@ -629,7 +637,14 @@ Because lower is better, these functions are sometimes called
...
+Trong học máy, hoặc nói rộng ra là lĩnh vực tối ưu hoá, ta gọi chúng là các hàm mục tiêu (_objective function_).
+Theo quy ước, ta thường định nghĩa các hàm tối ưu sao cho giá trị càng thấp thì mô hình càng tốt.
+Đấy chỉ là một quy ước ngầm.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Đây chỉ là một quy ước ngầm.
```

File: chapter_introduction/index_vn.md [hunk 12]
```diff
@@ -629,7 +637,14 @@ Because lower is better, these functions are sometimes called
...
+Bạn cũng có thể lấy một hàm $f$ sao cho giá trị càng cao thì càng tốt,
+sau đó lấy một hàm $f' = -f$ tương đương với giá trị càng thấp thì mô hình càng tốt.
+Chính vì ta mong muốn hàm có giá trị thấp, nó còn được gọi dưới tên *hàm mất mát* (_loss function_) và *hàm phí tổn* (_cost function_)
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Chính vì ta mong muốn hàm có giá trị thấp, nó còn được gọi dưới tên *hàm mất mát* (_loss function_) và *hàm chi phí* (_cost function_)
```

Anh đề xuất gọi "cost function" là hàm chi phí. Từ "tổn" nghe có vẻ negative.
